bootstrapper/bootstrapper.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+
+if [ -f "$STEP_ROOT" ];
+then
+    echo "Found existing $STEP_ROOT, skipping bootstrap"
```

Comment:
Does this mean that it will not run bootstrapper, if it already exists? 

Should the subject be 
Do not run bootstrapper on subsequent runs if already running?

Reply:
I'm happy to change this to match whatever messaging the project maintainers would prefer, I'd like t get feedback from them before making further changes though.